GoogleChromeLabs/ps-analysis-tool
# Feature: Show selected frame cookies in cookie table.

## Description

After the sidebar was created with vertical menu items, we had to show the cookies used by the frames from the vertical menu to the cookie listing in the cookie table.

## Relevant Technical Choices

Once the frame is selected, we check which cookies are present in that frame and calculate the list. Once the list is calculated it is then sent to the cookie table for listing them.

## Testing Instructions
1. Open the terminal in the root directory of the project.
2. Run the command npm run dev, and the extension's Webpack development will spin up.
3. Open the website `aljazeera.com`, open the DevTools panel and select the Privacy Sandbox tab.
4. In the sidebar, click on the arrow icon to expand the cookies section.
5. Select any frame  under there, it should show the cookies used by the frame in the table on the right side

## Screenshot/Screencast

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/59614577/9c33710f-3dd3-48c5-ac4d-c8cb4e4022ac

---


Partially addresses #71
